===> hello: Tests ready to run
=> Testing "hello" with args [] of type [] and result :...
=> "hello" tested with SUCCESS
=> Testing "testRound" with args [] of type [] and result :...
=> "testRound" tested with SUCCESS
=> Testing "testMin" with args [7, 3] of type [int, int] and result 3:int...
=> "testMin" tested with SUCCESS
=> Testing "testMin1" with args [7, 3] of type [int, int] and result 7:int...
=> "testMin1" tested but FAILED
=> Testing "isLarger" with args [7, 3] of type [int, int] and result false:bool...
=> "isLarger" tested but FAILED
=> Testing "isLarger1" with args [7, 3] of type [int, int] and result true:bool...
=> "isLarger1" tested with SUCCESS
=> Testing "idDouble" with args [3.14] of type [double] and result 3.14:double...
=> "idDouble" tested with SUCCESS
=> Testing "fail1" with args [3.14] of type [double] and result 3.14:double...
=> "fail1" not tested: wrong argument list
=> Testing "fail2" with args [3.14] of type [double, int] and result 3.14:double...
=> "fail2" not tested: wrong argument list
=> Testing "fail3" with args [3.14] of type [double] and result 3:int...
=> "fail3" not tested: wrong result type
=> Testing "fail4" with args [3.14] of type [double] and result 3:int...
=> "fail4" not tested: wrong result type
